onnxscript/values.py

```diff
@@ -294,8 +294,8 @@ def __call__(self, *args, **kwargs):
         # FIXME(after #225): Move import to the top of the file.
         from onnxscript import evaluator  # pylint: disable=import-outside-toplevel
 
-        new_args, has_array = _adapt_to_eager_mode(args)
-        result = evaluator.default().eval_function(self, *new_args, **kwargs)
+        # new_args, has_array = _adapt_to_eager_mode(args)
```

Comment:
Remove this line maybe?

Reply:
Where is the "adapt" happening now? I don't see it in `eval_function`. It has to happen somewhere (for the normal evaluators).

Reply:
I moved it to the Evaluator in https://github.com/microsoft/onnx-script/pull/352

Reply:
Done